pyproject.toml

```
@@ -48,6 +48,19 @@ select = [
     "E", # pycodestyle Error
     "F", # Pyflakes
     "W", # pycodestyle Warning
+    # Most refurb rules are in preview and can be opinionated,
+    # consider them individually as they come out of preview (last check: 0.8.4)
+    "FURB105", # Unnecessary empty string passed to `print`
+    "FURB129", # Instead of calling `readlines()`, iterate over file object directly
+    "FURB136", # Replace `if` expression with `{min_max}` call
+    "FURB161", # Use of `bin({existing}).count('1')`
+    "FURB163", # Prefer `math.{log_function}({arg})` over `math.log` with a redundant base
+    "FURB167", # Use of regular expression alias `re.{}`
+    "FURB168", # Prefer `is` operator over `isinstance` to check if an object is `None`
+    "FURB169", # Compare the identities of `{object}` and None instead of their respective types
+    "FURB177", # Prefer `Path.cwd()` over `Path().resolve()` for current-directory lookups
+    "FURB181", # Use of hashlib's `.digest().hex()`
+    "FURB187", # Use of assignment of `reversed` on list `{name}`
```

FURB161, FURB163, maybe FURB181: I don't think these are code typeshed would ever need. I wouldn't *disable* them if the category was enabled (harmless). But I don't think we need to *enable* them if listing rules individually (redundant).
```suggestion
    # Most refurb rules are in preview and can be opinionated,
    # consider them individually as they come out of preview (last check: 0.8.4)
    "FURB105", # Unnecessary empty string passed to `print`
    "FURB129", # Instead of calling `readlines()`, iterate over file object directly
    "FURB136", # Replace `if` expression with `{min_max}` call
    "FURB167", # Use of regular expression alias `re.{}`
    "FURB168", # Prefer `is` operator over `isinstance` to check if an object is `None`
    "FURB169", # Compare the identities of `{object}` and None instead of their respective types
    "FURB177", # Prefer `Path.cwd()` over `Path().resolve()` for current-directory lookups
    "FURB187", # Use of assignment of `reversed` on list `{name}`
```

---

I don't mind them being enabled, as they seem like reasonable rules, but I also don't mind them being disabled, since I agree that it's unlikely they'd ever be relevant and it would simplify our config a bit. I'm easy either way; happy to defer to you!